src/modules/getCommentItems.js

```diff
@@ -0,0 +1,9 @@
+import { baseUrl, involvementApi } from '../../config/keys.js';
+
+export default async (item) => {
+  const response = await fetch(
+    // eslint-disable-next-line
```

Comment:
- [ ] Enforcing linters in our project helps us write clean and consistent code. Kindly remove the line you used to disable linter checks. 😎️